frontend/hub/remote-registries/RemoteRegistryForm.tsx

```diff
@@ -69,6 +71,75 @@ export function CreateRemoteRegistry() {
   );
 }
 
+export function EditRemoteRegistry() {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+  const params = useParams<{ id: string }>();
+  const name = params.id;
+  const { data, error, refresh } = useGet<HubItemsResponse<RemoteRegistryProps>>(
+    hubAPI`/_ui/v1/execution-environments/registries/?name=${name ?? ''}`
```

Comment:
can name actually be empty? (I'm guessing yes since it comes from the page url)

filtering by `?name=` will return all registries, so this would start editing a random registry if name is undefined

Reply:
Yeah, that is a standard that we have been using in the code basis. As we talk we could update the apiTag to deal with that one.